Media/doc/introduction.md

```diff
@@ -0,0 +1,10 @@
+# Media Data Models
```

Comment:
I am not sure about the naming of the model. This about EventDetection or something similar. For me, media is about movies and so on :) Said so, I must admit, that i don't have any brilliant name coming to my mind.

Reply:
Well, these Data models are related with the concept of media described here
http://fiwaretourguide.readthedocs.io/en/latest/real-time-processing-of-media-streams/introduction/
So we think, these names fit with that.

Reply:
I understand where it comes from but then again MediaEvent would be more appropriate, since you are not modelling the Media and I believe very similar events could be media unrelated (you could detect an object, with no media involvement, for example with RFID). I think opening up the model to a more wide interpretation the Kurento one, is helping to adopt it. But this is my feeling :) @jmcanterafonseca is the real master of ceremony here.